guggero/lightning-terminal
# Add Pool page with Account Balances and Order Form sections

This PR adds a base skeleton of the Pool page along with the account balances and order form sections.

Edit: added order form section to this PR instead of creating a new one.

### Steps to test

1. Launch the app
1. Create an account if you don't already have one
   `store.accountStore.createAccount(2000000, 2016)`
1. Click on the Lightning Pool link in the left nav menu
1. Confirm you can see the Pool page with the account balances
1. Submit some orders using the order form

![image](https://user-images.githubusercontent.com/1356600/94355342-69a98380-0051-11eb-984f-07f7896b4d76.png)
